### DISCUSSION TITLE
Anyone else experiencing blippy Firefox context menus with regolith-compositor-picom-glx?

### DISCUSSION THREAD

**Discussion by jc00ke** [regolith-linux/regolith-desktop#801] [General]
Firefox 108.0.1, Regolith 2.2, and I had the latest version of `regolith-compositor-picom-glx` installed. None of the windows stayed open long after opening any context menu (right mouse click on page, right mouse click on tab, left mouse click on extension).

I've switched back to `regolith-compositor-none` for now and I've got my context menus back.

**Comment by kgilmer** [comment]
Hmm, I have not noticed this.  I do recall there are specific picom configuration options for context menus in it's config file.  It may be that something there needs to be updated.
